go.sum

```diff
@@ -234,6 +234,8 @@ github.com/kr/text v0.2.0 h1:5Nx0Ya0ZqY2ygV366QzturHI13Jq95ApcVaJBhpS+AY=
 github.com/kr/text v0.2.0/go.mod h1:eLer722TekiGuMkidMxC/pM04lWEeraHUUmBw8l2grE=
 github.com/kulti/thelper v0.2.0 h1:0hWMFWREtbyuGTPl/d2c1Kq/VfzNM97NzRwp1zMLaAc=
 github.com/kulti/thelper v0.2.0/go.mod h1:vMu2Cizjy/grP+jmsvOFDx1kYP6+PD1lqg4Yu5exl2U=
+github.com/kulti/thelper v0.2.1 h1:H4rSHiB3ALx//SXr+k9OPqKoOw2cAZpIQwVNH1RL5T4=
```

Comment:
please run `go mod tidy` it should clean up the `go.sum` from `thelper v0.2.0` lines

Reply:
Fixed